### DISCUSSION TITLE
如何默认使用某个vf滤镜，以及着色器一直不加载

### DISCUSSION THREAD

**Discussion by jerry460** [hooke007/mpv_PlayKit#337] [mpv 通用]
控制台没有任何报错就不截图了，着色器无论怎么设置在shirt+i后出现的视图中完全不加载（或许是加载了但是不显示）
纯小白，不知道如何默认启用某个vf滤镜，哪位大佬可以指点一下吗。使用的是最新版的mpv_lazy包，不知道如何修改mpv.config文件
以及我看超分辨率vf滤镜默认超到2k，怎么改成4呢

**Comment by hooke007** [comment]
你这问题又多又简单。我的建议是从头学所有操作

https://hooke007.github.io/unofficial/mpv_start.html
https://hooke007.github.io/unofficial/mpv_shaders.html
https://hooke007.github.io/unofficial/mpv_filters.html

**Reply by jerry460** [reply]
好的，想偷懒结果没偷成QAQ/笑哭。感谢大佬的回复
